### PR TITLE
Use ComplexExpresionBuilder to rewrite global variable

### PR DESCRIPTION
**Pull request by ptomin**
- Create `TypedConstantRewriter` test with dereferenced first struct field.

Expected:
```
        globals->t100100.dw0000
```
but was:
```
        globals->t100100
```

### CODE REVIEW

File: subjects/PE/x86/VCExeSample/VCExeSample.reko/VCExeSample_text.c [hunk 1]
```diff
@@ -134,9 +134,9 @@ void nested_structs_test13(nested_structs_type * str)
 void gbl_nested_structs_test14()
 {
 	// gbl_nested_structs.a = 5
-	globals->gbl_nested_structs = (Eq_n) 0x05;
+	globals->gbl_nested_structs.a = 0x05;
 	// gbl_nested_structs.str.b = 6
-	globals->gbl_nested_structs.str.b = (Eq_n) 0x06;
+	globals->gbl_nested_structs.str.b = 0x06;
```
Comment (ptomin) [inline subjects/PE/x86/VCExeSample/VCExeSample.reko/VCExeSample_text.c:139]: Now it looks almost ideally!

**Comment by ptomin** [comment]
There is Travis run in the list of executed checks again. Good!

**Comment by uxmal** [comment]
Yes, I was able to get Travis-CI reintegrated. Nice improvements, Pavel!

Any opinons on PR #870 - in particular the change to C# 8? If you are using Visual Studio, it will require you to upgrade to VS2019.

**Comment by ptomin** [comment]
I'll try to install `VS2019` and test PR #870 tomorrow

**Pull request merged**